ONSdigital/design-system
# Upgrade @babel/preset-env to remove caniuse-lite warning

### What is the context of this PR?
There is this warning when running DS. This PR updates @babel/preset-env to remove this warning
![image](https://user-images.githubusercontent.com/42928680/109990616-79fd1300-7d01-11eb-9608-03fb2d76222f.png)

### How to review 
- Check travis run from another branch, search for "caniuse" and see warning
- Check the travis run on this branch, and search for "caniuse" and see warning is now gone
- Tests pass
- Percy passes
